spec/features/home_page_spec.rb

```diff
@@ -36,7 +36,7 @@
 
       within('footer') do
         expect(page).to have_selector('h3', text: 'ScholarSphere')
-        expect(page).to have_content('A service of the University Libraries.')
+        expect(page).to have_content(I18n.t!('footer.description'))
```

Comment:
Ah, I did not know you could do this (`I18n.t!`).  Raise an error if the key is missing.  Good for tests.